pyproject.toml

```diff
@@ -7,13 +7,11 @@ license = "BSD-3"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = ">=3.9,<3.13"
```

Comment:
is it possible to get this to work with `^3.9`.

Reply:
At the moment no due to xarray. The dependencies couldn't be resolved when using `^3.9` with the `xarray` version in `SpatialData`

Reply:
or maybe let me retry because this was before merge

Reply:
no due to following:
```
Check your dependencies Python requirement: The Python requirement can be specified via the `python` or `markers` properties
    
    For xarray-dataclasses, a possible solution would be to set the `python` property to ">=3.9,<3.13"
```